crates/mipsy_interactive/src/interactive/runtime_handler.rs

```diff
@@ -76,29 +76,46 @@ where
 }
 
 fn get_input_int(name: &str, verbose: bool) -> Option<i32> {
-    let prompt: Box<dyn Fn()> = 
+    let bad_input_prompt: Box<dyn Fn()> =
```

Comment:
Now that I think about it, this can probably just be a `&dyn Fn()` if we move the `if` inside the closure:

```
    let bad_input_prompt: &dyn Fn() = &|| {
        if verbose {
            prompt::error_nonl(format!("bad input (expected {}), try again: ", name));
        } else {
            print!("[mipsy] bad input (expected {}), try again: ", name);
        }
    };
```